### ISSUE TITLE
github pages for basic docs and homepage?

### ISSUE THREAD

**Issue by CuddleBear92** [game-list/game-list#96]
How about using github pages for a singple one page homepage with a few screenshots, a download button, links to different places and a sinple page for basic docs?

https://pages.github.com/

Super simple to setup and is self hosted on github. perfect for simple things like startup guides and an simple overview.

**Comment by game-list** [comment]
Yes, that's a good idea, I'll try to do it when I have some time.
